transfermonitor/transfermonitor.go

```diff
@@ -94,6 +96,15 @@ func (t *TransferMonitor) Start(ctx context.Context) error {
 					if quickStartBlockHeight > startBlockHeight {
 						startBlockHeight = quickStartBlockHeight
 					}
+
+					// Mark this chain as having been quickstarted
+					t.didQuickStart[chainID] = true
+				} else {
+					if t.didQuickStart[chainID] {
+						// continue using last processed block height
+					} else {
+						lmt.Logger(ctx).Info("Started in non-quickstart mode")
+					}
```

Comment:
this is going to log every loop right after the initial quick start logic executes or if quick start is not used right? we should just log various flag values that we care about on startup

Reply:
right good point, i removed this log and added a log to print config and some flags on startup